### PR TITLE
develop update

### PR DESCRIPTION
**Pull request by bohuie**

### CODE REVIEW

File: skills_DS/api/views.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+from rest_framework import status
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework.parsers import FileUploadParser
+
+# Create your views here.
+class AnswersView(APIView):
+	def post(self, request, format=None):
+		if request.data:
+			print(request.data['age'], request.data['gender'], request.data['yearOfStudy'])
```
Comment (bohuie) [inline skills_DS/api/views.py]: The print statements in this file are unnecessary

File: skills_DS/api/views.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+			return Response({'hey': 'it worked'}, status=status.HTTP_200_OK)
```
Comment (bohuie) [inline skills_DS/api/views.py]: If you write code that is intended to change later, leave TODO comments to make that intent obvious

File: skills_DS/base/admin.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+from django.contrib import admin
+
+# Register your models here.
+
+from .models import User
+
+@admin.register(User)
+class CustomUserAdmin(admin.ModelAdmin):
```
Comment (bohuie) [inline skills_DS/base/admin.py:12]: Why is this here and not in models.py?

File: skills_DS/react/src/components/Questions.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+        setReceived(true)
+      })
+      .catch(err => {
+        console.error(err, err?.response, err?.response?.data)
+        setError("bad request")
+      })
+  }
+
+  function getCookie(name) {
+    let cookieValue = null
```
Comment (bohuie) [inline skills_DS/react/src/components/Questions.js]: Why do we need cookies and how are we using them?

**Review by brianzhouzc — APPROVED** [review]
All known bugs have been resolved.
